articles/install-guide/update.md

```diff
@@ -46,7 +46,7 @@ Verify the output is `3.0.100` or higher. If not, install the [latest version](h
 	```bash
 	dotnet add [project_name].csproj package Microsoft.Quantum.Development.Kit
 	```
-	If your projects use any other Microsoft.Quantum packages, run the command for these too.
+	If your projects use any other Microsoft.Quantum packages, run the command for these too. 
```

Comment:
Would like an example or two of what these packages might be. @bromeg suggested Microsoft.Quantum.Chemistry, but we wanted to get your input.

We definitely want to avoid inexperienced users thinking this means they have to run it for every namespace they use in each project (Microsoft.Quantum.Intrinsic, etc.). I was thinking an "(e.g. ...)" here might be helpful, or some way of making this less ambiguous to the inexperienced.

Reply:
My suggestion would be M.Q.Numerics rather than Chemistry, as Chemistry sometimes has other out-of-band dependencies (NWChem); that way, Numerics presents a simpler story. More generally, we currently don't have a way to indicate in API references which package provides a particular function, operation, or UDT. That would go a long way towards addressing that confusion.

Reply:
That makes sense, I'll use that one. Thanks!

Agreed---a bit of a pitfall to having such a breadth of functionality, I suppose.